### PR TITLE
Migrate to new Mapbox Static Tiles API

### PR DESCRIPTION
**Pull request by gherceg**
Related to this [JIRA ticket](https://dimagi-dev.atlassian.net/browse/SAAS-11243). As part of creating our own SaaS Mapbox account, any newly created accounts do not have access to legacy APIs. This migration had to happen anyway so it's good it was brought to our attention via via splitting of accounts. 
##### SUMMARY

When making a Mapbox API request with access tokens generated via the new account, a `410` response was returned because "Classic styles are no longer support for your account". More details [here](https://docs.mapbox.com/api/maps/#response-retrieve-raster-tiles-from-styles). 

Mapbox provides an easy to follow guide to migrate from a classic style to a new supported API [here](https://docs.mapbox.com/help/troubleshooting/migrate-legacy-static-tiles-api/?/=blog&utm_source=mapbox-blog&utm_campaign=blog%7Cmapbox-blog%7Cdoc-migrate-static%7Cdeprecating-studio-classic-styles-d8892ac38cb4-20-03&utm_term=doc-migrate-static&utm_content=deprecating-studio-classic-styles-d8892ac38cb4). 

To summarize the changes, the resource IDs changed from `mapbox.streets -> mapbox/streets-v11` and `mapbox.satellite -> mapbox/satellite-streets-v11`, and the URL changed slightly. I also added `tileSize=512` and `zoomOffset=-1` as the documentation recommends. I did notice without tileSize specified the map's text was smaller than usual, and the zoomOffset was needed to load the correct default position. 

Once this is merged I can rollout updated access tokens on all SaaS environments.

### CODE REVIEW

**Review by orangejenny — APPROVED** [review]
The beautiful PR description made this easy to review. Looks good.

**Review by dannyroberts — APPROVED** [review]

**Review by orangejenny — APPROVED** [review]

**Review by orangejenny — APPROVED** [review]
